app/components/explosions/explosions.html.erb

```diff
@@ -0,0 +1,14 @@
+<section class="grid">
+  <div class="line">
+  <% (0..@size*@size-1).each_with_index do |coord, i| %>
```

Comment:
could the range here be a method in the component class?

Reply:
Makes sense to me.